docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+
+(Optional) If you wish to build the documentation locally, you can install ``graphviz`` using the `Chocolatey <https://chocolatey.org/>`_ package manager,
+
+.. code:: bash
+
+	choco install graphviz --version=8.0.5
+
+or follow the instructions on the `graphviz website <https://graphviz.org/download/>`_.
+
+To install PyBaMM, you will need to install the following components:
+
+C++ Compiler
+~~~~~~~~~~~~
+
+PyBaMM uses a recent version of Microsoft Visual C++ (MSVC), which you can get using `Build Tools for Visual Studio Code 2022 <https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022/>`_.
+
+Note that you won't need Visual Studio 2022 entirely; just ``Desktop development with C++`` will suffice.
+
+.. image:: images-windows-source/pic1-msvc-select.png
+	:height: 456px
+	:width: 810px
+	:scale: 100%
+	:alt: Image with the packages to install
+
+
+VCPKG
+~~~~~
+
+PyBaMM comes with a DAE solver based on the IDA solver provided by the SUNDIALS library.
+To use this solver, you must ensure you have the necessary SUNDIALS components
+installed on your system.
+
+VCPKG automatically installs them for you during the build process. To learn more, look at the `official documentation by Microsoft <https://learn.microsoft.com/en-us/vcpkg/get_started/overview/>`_.
+
+To install VCPKG, follow the steps below:
+
+1. Open a Command Shell, and navigate to ``C:\``.
+    * We are installing VCPKG in ``C:\vcpkg``
+    * If you wish to install in a custom directory, change the environment variable ``VCPKG_ROOT_DIR`` before installation as mentioned herein :ref:`installing-pybamm`.
+2. Clone VCPKG's GitHub repository
+
+   .. code:: bash
+
+	  git clone https://github.com/microsoft/vcpkg.git
+
+
+   * If you do not have Git installed, download and install it from the `official Git download page <https://git-scm.com/download/win>`_.
+
+3. Navigate inside VCPKG directory and install it:
+
+   .. code:: bash
+
+	  cd vcpkg && bootstrap-vcpkg.bat
+
+4. Add VCPKG directory to ``Path`` environment variable:
+    * Copy the location ``C:\vcpkg`` or the location of your custom directory.
+    * In the start menu, search for ``Edit the system environment variables`` and open it.
+    * Under the ``Advanced`` tab, click on ``Environment Variables``
+        .. image:: images-windows-source/pic2-env-var-open.png
+			:scale: 70%
+
+
+    * Select ``Path`` from ``user variables`` if you want to set VCPKG just for you, or use ``Path`` from ``system variables`` to set VCPKG for all your system users.
+    	.. image:: images-windows-source/pic3-show-path.png
+		    :scale: 50%
+
+
+    * Select ``Path`` and select ``NEW``. Paste the location to VCPKG, then confirm your options by clicking ``OK``.
+    	.. image:: images-windows-source/pic4-save-env-var.png
+		  :height: 900px
+		  :width: 900px
+		  :scale: 70%
+
+
+5. Close and reopen Command Prompt and execute ``vcpkg`` to see an output of different options available for ``vcpkg``.
```

Comment:
Can we instead tell them to follow the instructions on the vcpkg git repo? We don't want to have to change this if the recommendations from microsoft change.

Reply:
Yes, we can do that. I will be removing this section. As you said, we can ask the users to install VCPKG and update their PATH permanently.